README.md

```diff
@@ -326,17 +353,19 @@ echo "gruntwork-io/terragrunt gruntwork-io/terratest" | git-xargs \
-| `--dry-run`              | If you are in the process of testing out `git-xargs` or your initial set of targeted repos, but you don't want to make any changes via the Github API (pushing your local changes or opening pull requests) you can pass the dry-run branch. This is useful because the output report will still tell you which repos would have been affected, without actually making changes via the Github API to your remote repositories. | Boolean | No       |
-| `--max-concurrent-repos` | Limits the number of concurrent processed repositories. This is only useful if you encounter issues and need throttling when running on a very large number of repos. Default is `0` (Unlimited)                                                                                                                                                                                                                                | Integer | No       |
+| Flag                     | Description                                                                                                                                                                                                                                                                                                                                                                                                                    | Type    | Required |
+| ------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------- | -------- |
+| `--branch-name`          | You must specify the name of the branch to make your local and remote changes on. You can further control branching behavior via `--skip-pull-requests` as explained below                                                                                                                                                                                                                                                     | String  | Yes      |
+| `--loglevel`             | Specify the log level of messages git-xargs should print to STDOUT at runtime. By default, this is INFO - so only INFO level messages will be visible. Pass DEBUG to see runtime errors encountered by your scripts or commands. Accepted levels are TRACE, DEBUG, INFO, WARNING, ERROR, FATAL and PANIC.                                                                                                                      | String  | Yes      |
```

Comment:
NIT: I would put `Default: INFO` at the very end of this description the way we do in other places, and like you did in `max-concurrent-retries`. That makes it a bit easier to scan for.

Reply:
Makes sense! I also added the Default value to the end of the description in b6e44b8 because it's a common source of confusion for users, so I think it's worth overstating. Also removed a duplicated table row.